sympy/series/tests/test_series.py

```diff
@@ -29,6 +29,12 @@ def test_exp2():
     assert e1 == e2
 
 
+def test_asymptotic_series():
```

Comment:
This looks like the same test as above in test_aseries. Maybe that alone would suffice?

Reply:
This test is needed as the call here is to `series(x, oo)`, which raises `PoleError` on master (which doesn't call) `aseries`. Perhaps the test case in test_aseries can be removed

Reply:
Ok.